packages/selenium/src/AxeBuilder.cs

```diff
@@ -21,13 +22,26 @@ public class AxeBuilder
         private readonly AxeRunContext runContext = new AxeRunContext();
         private AxeRunOptions runOptions = new AxeRunOptions();
         private string outputFilePath = null;
+        private bool useLegacyMode = false;
+        private bool disableIframeTesting = false;
```

Comment:
This is already tracked as `runOptions.Iframes`, no? I think adding a new builder method to make it easier to use makes sense, but there should only be one source of truth for the data